EugeneMunya/QuickCredit3
# #166184545 view spesific loan application

#### What does this PR do?
Adding retrieve a single specific loan application from the database functionality in the system. 
#### Description of Task to be completed?
At this point retrieving specific loan application in the database is possible.
#### How should this be manually tested?
After cloning this repo and running npm start, then open postman and use a link looks like this localhost:3000/api/v1/loans/1 but make sure that you have logged in.
and pass your token as an authorization header.
#### What are the relevant pivotal tracker stories?
#166184545
#### Screenshots
![db](https://user-images.githubusercontent.com/36619897/58249696-dc209a00-7d5f-11e9-8944-0fb23f2a957e.jpeg)


#### Questions
No questions
